### ISSUE TITLE
sails.config.globals missing 'adapters'

### ISSUE THREAD

**Issue by tarlepp** [balderdashy/sails-docs#564]
I just noticed that globals configuration is missing adapters. I have disabled all but ```sails``` in this file and didn't see any reference to this ```adapters```.

```
sails> console.log(sails.config.globals);
{ _: false,
  async: false,
  sails: true,
  services: false,
  models: false,
  adapters: true }
```

There was a previous issue #451 about this too but @sailsbot closed that one and I can't re-open that one anymore.

Should those issues to be re-opened if any progress haven't occurred? Imho it's dummy to make new issues after bot closes those...

**Comment by animedbz16** [comment]
@tarepp:

I'm guessing this issue wasn't discussed since it doesn't really deal with the sails-docs repo, which is used for building documentation for the web site.

I just tested this on latest node (4.1.1) and sails (0.11.2)

```
[user@server ~]$ which node
/home/user/.nvm/versions/node/v4.1.1/bin/node
[user@server ~]$ which sails
/home/user/.nvm/versions/node/v4.1.1/bin/sails
[user@server ~]$ sails --version
0.11.2
[user@server ~]$ cd app/
[user@server app]$ sails console

info: Starting app in interactive mode...

info: Welcome to the Sails console.
info: ( to exit, type <CTRL>+<C> )

sails> console.log(sails.config.globals);
{ adapters: true, models: true, services: true }
undefined
sails> 
```

Looking through the code that generates the default globals.js config file, it appears to be in the following repository:
https://github.com/balderdashy/sails-generate-backend/blob/master/templates/config/globals.js

My guess is that an issue / PR to this repo would be a better place to correct the issue.

**Comment by tarlepp** [comment]
actually this is issue to both of those repositories (this and sails-generate-backend). I'll try to make PR to each of those.

**Comment by sailsbot** [comment]
Thanks for posting, @tarlepp.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
